packages/web/pages/apps/index.tsx

```diff
@@ -0,0 +1,168 @@
+import Fuse from "fuse.js";
+import React from "react";
+import { useTranslation } from "react-multi-lang";
+
+import { Button } from "~/components/buttons";
+import { HeroCard } from "~/components/cards";
+import AppDisplayCard from "~/components/cards/app-display-card";
+import { SearchBox } from "~/components/input";
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+
+type AppDataType = {
+  title: string;
+  subtitle: string;
+  thumbnail_image_URL: string;
+  external_URL: string;
+  twitter_URL?: string;
+  github_URL?: string;
+  medium_URL?: string;
+  featured?: boolean;
+};
+
+type App = {
+  title: string;
+  subtitle: string;
+  external_URL: string;
+  thumbnail_image_URL: string;
+  hero_image_URL: string;
+  twitter_URL?: string;
+  medium_URL?: string;
+  github_URL?: string;
+  featured?: boolean;
+};
+
+type AppStoreProps = {
+  apps: {
+    applications: App[];
+  };
+};
+
+export const AppStore: React.FC<AppStoreProps> = ({ apps }) => {
+  const [searchValue, setSearchValue] = React.useState<string>("");
+  const [fuzzySearchResults, setFuzzySearchResults] = React.useState<
+    AppDataType[]
+  >([]);
+  const [fuse, setFuse] = React.useState<Fuse<App> | null>(null);
```

Comment:
Nit: I'd prefer if we used `useState` instead of `React.useState` to keep consistency within the code base.